contrib/seeds/makeseeds.py

```diff
@@ -111,54 +115,54 @@ def filtermultiport(ips):
-        return asn
-    except Exception:
-        sys.stderr.write('ERR: Could not resolve ASN for "' + ip + '"\n')
-        return None
+# def lookup_asn(net, ip):
+#     '''
+#     Look up the asn for an IP (4 or 6) address by querying cymru.com, or None
+#     if it could not be found.
+#     '''
+#     try:
+#         if net == 'ipv4':
+#             ipaddr = ip
+#             prefix = '.origin'
+#         else:                  # http://www.team-cymru.com/IP-ASN-mapping.html
+#             res = str()                         # 2001:4860:b002:23::68
+#             for nb in ip.split(':')[:4]:  # pick the first 4 nibbles
+#                 for c in nb.zfill(4):           # right padded with '0'
+#                     res += c + '.'              # 2001 4860 b002 0023
+#             ipaddr = res.rstrip('.')            # 2.0.0.1.4.8.6.0.b.0.0.2.0.0.2.3
+#             prefix = '.origin6'
+#
+#         asn = int([x.to_text() for x in dns.resolver.query('.'.join(
+#                    reversed(ipaddr.split('.'))) + prefix + '.asn.cymru.com',
+#                    'TXT').response.answer][0].split('\"')[1].split(' ')[0])
+#         return asn
+#     except Exception:
+#         sys.stderr.write('ERR: Could not resolve ASN for "' + ip + '"\n')
+#         return None
 
 # Based on Greg Maxwell's seed_filter.py
-def filterbyasn(ips, max_per_asn, max_per_net):
-    # Sift out ips by type
-    ips_ipv46 = [ip for ip in ips if ip['net'] in ['ipv4', 'ipv6']]
-    ips_onion = [ip for ip in ips if ip['net'] == 'onion']
-
-    # Filter IPv46 by ASN, and limit to max_per_net per network
-    result = []
-    net_count = collections.defaultdict(int)
-    asn_count = collections.defaultdict(int)
-    for ip in ips_ipv46:
-        if net_count[ip['net']] == max_per_net:
-            continue
-        asn = lookup_asn(ip['net'], ip['ip'])
-        if asn is None or asn_count[asn] == max_per_asn:
-            continue
-        asn_count[asn] += 1
-        net_count[ip['net']] += 1
-        result.append(ip)
-
-    # Add back Onions (up to max_per_net)
-    result.extend(ips_onion[0:max_per_net])
-    return result
+# def filterbyasn(ips, max_per_asn, max_per_net):
+#     # Sift out ips by type
+#     ips_ipv46 = [ip for ip in ips if ip['net'] in ['ipv4', 'ipv6']]
+#     ips_onion = [ip for ip in ips if ip['net'] == 'onion']
+#
+#     # Filter IPv46 by ASN, and limit to max_per_net per network
+#     result = []
+#     net_count = collections.defaultdict(int)
+#     asn_count = collections.defaultdict(int)
+#     for ip in ips_ipv46:
+#         if net_count[ip['net']] == max_per_net:
+#             continue
+#         asn = lookup_asn(ip['net'], ip['ip'])
+#         if asn is None or asn_count[asn] == max_per_asn:
+#             continue
+#         asn_count[asn] += 1
+#         net_count[ip['net']] += 1
+#         result.append(ip)
+#
+#     # Add back Onions (up to max_per_net)
+#     result.extend(ips_onion[0:max_per_net])
+#     return result
```

Comment:
This is still apart of Bitcoin Core, but I'm guessing that this is made redundant for VTC because of different `nodes_main.txt`? Is that correct?

Reply:
I used a dump from jk14's seeder so nodes_main.txt are vertcoin nodes.

Reply:
Okay, sounds good.